src/storj.c

```diff
@@ -1757,3 +1758,113 @@ STORJ_API int storj_bridge_register(storj_env_t *env,
     }
     return uv_queue_work(env->loop, (uv_work_t*) work, json_request_worker, cb);
 }
+
+
+STORJ_API int storj_list_buckets(storj_api_t *storj_api)
+{
+    storj_api->last_cmd_req  = NULL;
+    storj_api->curr_cmd_req  = "get-bucket-id-req";
+    storj_api->next_cmd_req  = NULL;
+    storj_api->final_cmd_req = NULL;
+    storj_api->excp_cmd_resp = "get-bucket-id-resp";
+
+    /* when callback returns, we store the bucket id of bucket name else null */
+    return storj_bridge_get_buckets(storj_api->env, storj_api, get_buckets_callback);
+}
+
+STORJ_API int storj_get_bucket_id(storj_api_t *storj_api)
+{
+    int ret = -1;
+    char *bucket_name = storj_api->bucket_name;
+    if (!bucket_name)
+    {
+        printf("Missing : <bucket-name>\n");
+        return ret;
+    }
+
+    storj_api->last_cmd_req  = NULL;
+    storj_api->curr_cmd_req  = "get-bucket-id-req";
+    storj_api->next_cmd_req  = NULL;
+    storj_api->final_cmd_req = NULL;
+    storj_api->excp_cmd_resp = "get-bucket-id-resp";
+
+    /* when callback returns, we store the bucket id of bucket name else null */
+    return storj_bridge_get_buckets(storj_api->env, storj_api, get_bucket_id_callback);
```

Comment:
This can be `storj_bridge_get_bucket_id`, which was recently added.

Reply:
It just happened to be a coincidence, I am open for renaming the function. For the time being I will leave there.

Reply:
It's an already existing function, see https://github.com/Storj/libstorj/blob/master/src/storj.c#L1451

Reply:
changed the function name to cli-get-bucket_id()